docs/notes/2.26.x.md

```diff
@@ -104,7 +104,9 @@ A bug in the Django backend has been fixed so that a repo may have no Django app
 
 [Pyright](https://www.pantsbuild.org/stable/reference/subsystems/pyright) was updated from version [1.1.383](https://github.com/microsoft/pyright/releases/tag/1.1.383) to [1.1.396](https://github.com/microsoft/pyright/releases/tag/1.1.396).
 
-The Python backend uses the PyPI `packaging` distribution to parse Python version numbers. In upgrading `packaging` from v21.3 to v24.2, support for parsing "legacy" Python versions was removed. We do not anticipate an issue here (since [PEP 440](https://peps.python.org/pep-0440/) has been around since 2013), but please contact the maintainers if an issue does arise.
+The Python backend uses the PyPI `packaging` distribution to parse Python version numbers and requirements instead of the deprecated `pkg_resources` distribution. In upgrading `packaging` from v21.3 to v24.2 and in removing uses of `pkg_resources`, several user-visible changes occurred:
+- There is no more support for parsing "legacy" Python versions. We do not anticipate an issue here (since [PEP 440](https://peps.python.org/pep-0440/) has been around since 2013), but please contact the maintainers if an issue does arise.
+- Requirements containing a version with a trailing `.*` pattern are only valid when used with the `==` or `!+` operators; for example, `==3.9.*` or `!=3.10.*`. Pants previously accepted such requirements when used with the other operators (e.g., `<=`, `<`, `>`, or `>=`) because `pkg_resources` accepted that syntax. The [current version specifiers specification](https://packaging.python.org/en/latest/specifications/version-specifiers/) does not allow that syntax and thus `packaging`'s requiremens parser does not either.
```

Comment:
```suggestion
- Requirements containing a version with a trailing `.*` pattern are only valid when used with the `==` or `!+` operators; for example, `==3.9.*` or `!=3.10.*`. Pants previously accepted such requirements when used with the other operators (e.g., `<=`, `<`, `>`, or `>=`) because `pkg_resources` accepted that syntax. The [current version specifiers specification](https://packaging.python.org/en/latest/specifications/version-specifiers/) does not allow that syntax and thus `packaging`'s requirements parser does not either.
```